# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-uuid.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

fl_wrapper.so: write outside basedir (/usr/src/t2-src): /var/lib/libuuid/clock.txt

--- Arctic-Fox-46.1/python/mach/mach/main.py.vanilla	2025-08-12 22:58:20.753951860 +0200
+++ Arctic-Fox-46.1/python/mach/mach/main.py	2025-08-12 22:58:26.283084680 +0200
@@ -260,7 +260,7 @@
                 mod = imp.new_module(b'mach.commands')
                 sys.modules[b'mach.commands'] = mod
 
-            module_name = 'mach.commands.%s' % uuid.uuid1().get_hex()
+            module_name = 'mach.commands.%s' % uuid.uuid4().get_hex()
 
         imp.load_source(module_name, path)
 
